frontend/CHANGELOG.md

```diff
@@ -22,6 +22,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Fixed combobox loader not working in muscleGroup selector
 - Fixed issue where weigh ins page showed an error if user had no weigh ins.
 - Fixed issue where you could not leave maxReps as 0.
+- Fixed Loader bug when selecting size medium
+- Fixed default diet plan loading for users with existing plans
+- Fixed workout plan presets caching invalidation issue when adding/updating workout plan preset.
+- Fixed error toast descriptions showing in users form
```

Comment:
When I look at this i think you fixed an issue that the toast was being shown in the user form but when i look at the code i see what the fix actually is. 

I'd write Fixed issue where the description in the error toast was not shown in the UserForm.